src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculator, Drawable {
+    private double basis1;
+    private double basis2;
```

Comment:
```suggestion
    private double firstBasis;
    private double secondBasis;
```
avoid using numbers in variable names